tests/test_lstm.py

```diff
@@ -775,6 +775,31 @@ def func(x):
             return tf.identity(y[0], name="output"), tf.identity(y[1], name="output1")
         self.run_test_case(func, {"input:0": x_val}, [], ["output:0", "output1:0"], rtol=1e-05, atol=1e-06)
 
+    @check_tf_min_version("2.0")
+    @skip_tf_versions("2.1", "Bug in TF 2.1")
```

Comment:
Which bug are we talking about here? Could you please put the url into the comment?

Reply:
no clue I copied the structure of the bigru test that had this decorator, do you think I can remove it?

Reply:
> no clue I copied the structure of the bigru test that had this decorator, do you think I can remove it?

Yes please, let's have a try.

Reply:
BTW, TF 2.1 has been removed out of the CI pipeline, so this won't be run anyway.

Reply:
done, waiting for the CI to complete